explore/src/main/scala/explore/Routing.scala

```diff
@@ -85,7 +87,8 @@ object Routing {
       import dsl._
 
       def id[Id](implicit gid: Gid[Id]): StaticDsl.RouteB[Id] =
-        string(gid.regexPattern).pmapL(gid.fromString)
+        // We can't use gid.regexPattern because capture groups don't work here.
```

Comment:
Is this what was causing trouble in URLs with multiple ids?

Reply:
Yep. That was the problem. Seemed to work with one id, probably when it was the last think in the url. But, failed to parse the url correctly when there was a second one.